### ISSUE TITLE
CMake deprecation warning

### ISSUE THREAD

**Issue by mlund** [pybind/pybind11_json#64]
On newer versions of cmake I get this warning:
```
CMake Deprecation Warning at CMakeLists.txt:9 (cmake_minimum_required):
  Compatibility with CMake < 3.5 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.
```

**Comment by ebenali** [comment]
probably just copy the header into your tree under 3rdparty/ or whatever

**Comment by mlund** [comment]
> probably just copy the header into your tree under 3rdparty/ or whatever

I can certainly do that, but the warning suggests that the current `CMakeLists.txt` will become outdated. An alternative solution would be to increase the minimum CMake version to 3.5, released in 2016.

**Comment by martinRenou** [comment]
Fixed by #65
